trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -39,286 +40,582 @@ def plot_vehicle(vehicle, save_figure = False, plot_control_points = True, save_
+                    prop_collection = Poly3DCollection(prop_verts)
+                    prop_collection.set_facecolor(prop_face_color)
+                    prop_collection.set_edgecolor(prop_edge_color) 
+                    prop_collection.set_alpha(prop_alpha)
+                    axes.add_collection3d(prop_collection) 
+    return 
+
+def generate_nacelle_points(VD,propulsor,propulsor_name):
+    """ This generates the coordinate points on the surface of the fuselage
+
+    Assumptions: 
+    None
+
+    Source:   
+    None
     
-                    # ------------------------------------------------------------------------
-                    # Plot Propeller Blade 
-                    # ------------------------------------------------------------------------
-                    prop_face_color = 'red'
-                    prop_edge_color = 'red'
-                    prop_alpha      = 1
-                    for sec in range(dim-1): 
-                        for loc in range(af_pts): 
-                            X = [G.XA1[sec,loc],
-                                 G.XB1[sec,loc],
-                                 G.XB2[sec,loc],
-                                 G.XA2[sec,loc]]
-                            Y = [G.YA1[sec,loc],
-                                 G.YB1[sec,loc],
-                                 G.YB2[sec,loc],
-                                 G.YA2[sec,loc]]
-                            Z = [G.ZA1[sec,loc],
-                                 G.ZB1[sec,loc],
-                                 G.ZB2[sec,loc],
-                                 G.ZA2[sec,loc]]                    
-                            prop_verts = [list(zip(X, Y, Z))]
-                            prop_collection = Poly3DCollection(prop_verts)
-                            prop_collection.set_facecolor(prop_face_color)
-                            prop_collection.set_edgecolor(prop_edge_color) 
-                            prop_collection.set_alpha(prop_alpha)
-                            axes.add_collection3d(prop_collection)   
-                    
-                
+    Inputs:   
+    VD                   - vortex distribution    
+    
+    Properties Used:
+    N/A
+    
+    
+    """      
+    
+    tessellation = 24  
+    
+    if (propulsor.tag == 'Battery_Dual_Propeller') or (propulsor.tag =='Lift_Cruise'):
+        if propulsor_name == 'propeller':
+            l = propulsor.propeller_engine_length
+            h = propulsor.propeller_nacelle_diameter/2 
+            origin = propulsor.propeller.origin
             
-    plt.axis('off') 
-    plt.grid(None)  
+        if propulsor_name == 'rotor':
+            l = propulsor.rotor_engine_length
+            h = propulsor.rotor_nacelle_diameter/2  
+            origin = propulsor.rotor.origin
+            
+    else: 
+        h = propulsor.nacelle_diameter/2
+        l = propulsor.engine_length
+        origin = propulsor.origin
+        
+    end    = propulsor.nacelle_end
```

Comment:
I don't think these are descriptive definitions. They're also not defaulted in a base class for the networks.

Reply:
nacelle geometry removed in this PR